build/esbuild-build.ts

```diff
@@ -53,6 +53,7 @@ function createEnvDefines(envVarNames: string[], extras: Record<string, unknown>
       throw new Error(`Missing environment variable: ${name}`);
     }
   }
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
Compiler directives are not a good sign. What's this

Reply:
Was because the `in` is forbidden but was safe to use as it is just a Record, can change to a `forEach`

Reply:
They are there for a reason so you should change to for each if that's the case.